threeML/classicMLE/joint_likelihood.py

```diff
@@ -283,6 +284,8 @@ def fit(self, quiet=False, compute_covariance=True):
 
         total = 0
 
+        total_number_of_data_points = 0
```

Comment:
You should add meaningful comments throughout the code you are committing to explain what is happening and why, so we will remember when we'll change the code years from now...

Normally it should be one line / one comment, or close to that ;-)